tests/e2e/users/add-user.specs.js

```diff
@@ -1,24 +1,41 @@
-const utils = require('../../utils');
-const usersPage = require('../../page-objects/users/users.po.js');
-const helper = require('../../helper');
-const addUserModal = require('../../page-objects/users/add-user-modal.po.js');
-
-const addedUser = 'fulltester' + new Date().getTime();
-const fullName = 'Bede Ngaruko';
-const errorMessagePassword = element(by.css('#edit-password ~ .help-block'));
+const utils=require('../../utils');
+const usersPage=require('../../page-objects/users/users.po.js');
+const helper=require('../../helper');
+const addUserModal=require('../../page-objects/users/add-user-modal.po.js');
+const { browser }=require('protractor');
+const addedUser='fulltester';
+const fullName='Full Tester';
+const errorMessagePassword=element(by.css('#edit-password ~ .help-block'));
 
 describe('Add user  : ', () => {
+  let originalTimeout;
+  const admin=element(by.xpath(`//*[contains(normalize-space(text()), "Administrator")]`));
```

Comment:
Can you move element definitions to page objects? Whether existing or new? Also, xpaths should be the last resort. Can you add attributes to the elements that help identify?